Sources/SwiftSyntaxBuilder/SyntaxNodeWithBody.swift

```diff
@@ -69,8 +69,10 @@ public protocol HasTrailingCodeBlock: WithCodeBlockSyntax {
   init(_ header: SyntaxNodeString, @CodeBlockItemListBuilder bodyBuilder: () throws -> CodeBlockItemListSyntax) rethrows
 }
 
-public extension HasTrailingCodeBlock where Self: StmtSyntaxProtocol {
-  init(_ header: SyntaxNodeString, @CodeBlockItemListBuilder bodyBuilder: () throws -> CodeBlockItemListSyntax) throws {
+extension HasTrailingCodeBlock where Self: StmtSyntaxProtocol {
+  public init(_ header: SyntaxNodeString, @CodeBlockItemListBuilder bodyBuilder: () throws -> CodeBlockItemListSyntax)
+    throws
```

Comment:
Could you format this to break at the arguments?

```suggestion
  public init(
    _ header: SyntaxNodeString, 
    @CodeBlockItemListBuilder bodyBuilder: () throws -> CodeBlockItemListSyntax
  ) throws
```